third_party/islet/setup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+ROOT=$(git rev-parse --show-toplevel)
```

Comment:
Is this better named as `CC_ROOT`, for clarity that this is the top-level git-sub-dir for CC library?

(Or, if this is some other repo, consider adding a prefix to indicate what this `root` dir is for.)

Reply:
I changed it to `CC_ROOT`.